src/VirtoCommerce.ReturnModule.Data/Services/ReturnService.cs

```diff
@@ -69,77 +69,74 @@ public override async Task<IReadOnlyCollection<Return>> GetAsync(List<string> id
             return returns;
         }
 
-        public virtual async Task<IEnumerable<string>> SaveChangesAsync(Return[] returns)
+        public override async Task<IList<string>> SaveChangesAsync(IEnumerable<Return> returns)
```

Comment:
We are overriding base method implementation, why don't we just call base one and then select result data?

Reply:
Base method does not allow to save entities with no Ids.